pkgs/development/python-modules/langgraph/default.nix

```diff
@@ -59,6 +69,20 @@ buildPythonPackage rec {
 
   disabledTests = [
     "test_doesnt_warn_valid_schema" # test is flaky due to pydantic error on the exception
+    # tests require database connection
+    "test_interruption_without_state_updates"
+    "test_no_modifier"
+    "test_interruption_without_state_updates_async"
+    "test_no_modifier_async"
+    "test_nested_graph_interrupts_parallel"
+    "test_nested_graph_interrupts"
+    "test_pending_writes_resume"
+    "test_fork_always_re_runs_nodes"
+    "test_remove_message_via_state_update"
+    "test_channel_values"
+    "test_invoke_two_processes_in_out_interrupt"
+    "test_cancel_graph_astream"
+    "test_cancel_graph_astream_events_v2"
```

Comment:
Please consider using `postgresqlTestHook`.

Reply:
I tried, but the tests are creating new databases. How can I set `postgresqlTestHook` to allow the creation of new databases?

Reply:
I only removed the tests requiring to create database with random names.